dumpling/export/dump.go

```diff
@@ -58,6 +59,17 @@ type Dumper struct {
 	charsetAndDefaultCollationMap map[string]string
 
 	speedRecorder *SpeedRecorder
+	bufChan       chan writeRes
+	wg            *errgroup.Group
+}
+
+func getTable(tables map[databaseName][]*TableInfo) (string, *TableInfo, error) {
```

Comment:
Does this function aim to return the first db's first table?

Reply:
No. This function only operates on one table.